CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### Features
```

Comment:
Can probably just add `MD024` to markdownlint-disable to get the lint to pass

Reply:
nice!